docs/spelling_wordlist

```diff
@@ -20,6 +20,7 @@ dialogs
 Django
 draggable
 Flexbox
+geolocation
```

Comment:
I'm not convinced we need to introduce the extra namespace. Yes, it would make the app class smaller, but at the cost of a longer path every time you want to do something hardware related. 

If we need a terminology distinction - a window has a Position, not a Location.

Plus, if it turns out I'm wrong, it's easy enough to add it later.